src/overlays/actors/ovl_En_Mu/z_en_mu.c

```diff
@@ -82,20 +82,20 @@ void EnMu_Interact(EnMu* this, PlayState* play) {
         }
     }
 
+    // If all 5 messages have been spoken, we reset but make sure the last message isn't repeated again
     if (i == 5) {
         if (this->defaultTextId == (textIdOffset[randomIndex] | 0x7000)) {
```

Comment:
I personally dislike writing comments with "we" and would prefer if it was removed. the sentence reads just fine without it.
But its not a big deal and will merge soon if it isnt addressed.